tests/guardrails/test_guardrails_types.py

```
@@ -191,131 +212,78 @@ def test_prefers_the_most_specific_source(self, result, expected):
+    messages in it, regex puts the matched text in it, and f5 forwards the provider
+    response. A rail that wants the caller to know why it blocked says so in ``reason``,
+    which it authors for exactly that purpose.
+    """
 
     CROWDSTRIKE_VERDICT = {
-        "allowed": False,
         "blocked": True,
         "guard_output": "policy 7",
         "user_message": "my private question",
         "bot_message": "the draft reply",
     }
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)